.travis.yml

```diff
@@ -14,26 +14,27 @@
 # limitations under the License.
 #
 
+# The prebuilt pypy packages at https://antocuni.github.io/ don't work with the older default dist
```

Comment:
```suggestion
# The prebuilt pypy packages at https://antocuni.github.io/pypy-wheels/ don't work with the older default dist
```
https://antocuni.github.io/ not a valid url, do you want to use https://antocuni.github.io/pypy-wheels/ instead?